.github/workflows/cross-version-tests.yml

```diff
@@ -90,9 +91,12 @@ jobs:
               issue_number: context.issue.number,
             });
             const labelNames = labels.data.map(({ name }) => name);
+            const { body } = context.payload.pull_request;
+            const flavorsMatch = body.match(/\/flavors\s+([^\n]+)/);
```

Comment:
Another option is labels. We can create labels shown below and attach them on the PR.

- flavors: sklearn
- flavors: tensorflow

which seems a bit inconvenient.